### PR TITLE
Fix compatibility issue and add exponential backoff.

### PR DESCRIPTION
**Pull request by TanYongSql**
At a minimum please provide the following:

### CODE REVIEW

**Review by songting-chen — APPROVED** [review]

File: tools/etl/tg-jdbc-driver/tg-jdbc-driver/src/main/java/com/tigergraph/jdbc/restpp/RestppConnection.java [hunk 1]
```diff
@@ -502,6 +504,13 @@ public RestppResponse executeQuery(QueryParser parser,
               ", payload size: " + json.length(), e);
           throw new SQLException("Failed to execute query: request: " + request +
               ", payload size: " + json.length(), e);
+        } else {
+          // Exponential Backoff
+          try {
+            Thread.sleep((long)(Math.pow(2, retry) * 5000));
+          } catch (InterruptedException ex) {
```
Comment (songting-chen) [inline tools/etl/tg-jdbc-driver/tg-jdbc-driver/src/main/java/com/tigergraph/jdbc/restpp/RestppConnection.java:511]: nit: add some logging here

File: tools/etl/tg-jdbc-driver/tg-jdbc-driver/src/main/java/com/tigergraph/jdbc/restpp/RestppConnection.java [hunk 2]
```diff
@@ -411,13 +411,15 @@ public RestppResponse executeQueries(final List<QueryParser> queries) throws SQL
...
   private void getToken() throws SQLException {
     StringBuilder urlSb = new StringBuilder();
-    // /gsqlserver/gsql/authtoken is deprecated, it won't return error messages when
-    // password is wrong
-    urlSb.append("/restpp/requesttoken");
     // If restpp version is under 3.5, pass graph name as a parameter
     if (this.restpp_version.compareTo(new ComparableVersion("3.5.0")) < 0) {
+      // /gsqlserver/gsql/authtoken is deprecated, it won't return error messages when
+      // password is wrong
+      urlSb.append("/gsqlserver/gsql/authtoken");
```
Comment (pxieTG) [inline tools/etl/tg-jdbc-driver/tg-jdbc-driver/src/main/java/com/tigergraph/jdbc/restpp/RestppConnection.java:418]: Is `/restpp/requesttoken` incompatible with 3.5-? I have tested `/gsqlserver/gsql/authtoken` with 3.4 and found it won't return error messages for wrong password.

Comment (TanYongSql) [reply tools/etl/tg-jdbc-driver/tg-jdbc-driver/src/main/java/com/tigergraph/jdbc/restpp/RestppConnection.java:418]: USBank uses v3.2.2, which doesn't support `/requesttoken`

**Review by pxieTG — APPROVED** [review]
Test:
3.2: request token(pass), exponential backoff(pass)
3.4: request token(pass), exponential backoff(pass)
3.5: request token(pass), exponential backoff(pass)

Note: The `/gsqlserver/gsql/authtoken` won't tell it if the password is wrong.

**Pull request merged**